composer.json

```diff
@@ -11,8 +11,9 @@
         }
     ],
     "require": {
-        "php": ">=5.4",
-        "psr/http-message": "^1.0"
+        "php": "^7.2",
```

Comment:
i'd prefer to still support 7.1 (its still security fixed for another year) to help people stuck on slow upgrading systems.